src/rez/utils/platform_.py

```diff
@@ -481,6 +481,10 @@ def symlink(self, source, link_name):
     def _terminal_emulator_command(self):
         return "CMD.exe /Q /K"
 
+    def _difftool(self):
+        from rez.util import which
+        return which("meld", "fc")
```

Comment:
This will conflict with a change in pull request #204 - however your change is better.  Hopefully @nerdvegas can fix it up when he does the merge of the pull requests.